dbatools.psm1

```diff
@@ -1102,15 +1102,31 @@ if ($option.LoadTypes -or
 #. Import-ModuleFile "$script:PSModuleRoot\bin\type-extensions.ps1"
 #Write-ImportTime -Text "Loaded type extensions"
 
-$td = (Get-TypeData -TypeName Microsoft.SqlServer.Management.Smo.Server)
-[Sqlcollaborative.Dbatools.dbaSystem.SystemHost]::ModuleImported = $true;
+
+
+# no idea what this is for
+$null = (Get-TypeData -TypeName Microsoft.SqlServer.Management.Smo.Server)
```

Comment:
Maybe just remove the line and see if someone complains?

Reply:
I can see if our tests complain